components/Gallery/ArticleGallery.js

```diff
@@ -32,7 +42,12 @@ const findFigures = (node, acc = []) => {
 
 const getImageProps = node => {
   const url = get(node, 'children[0].children[0].url', '')
+  const urlDark = get(node, 'children[0].children[2].url')
   const captionMdast = get(node, 'children[1].children', [])
+  const included =
+    !get(node, 'data.excludeFromGallery', false) &&
+    imageSizeInfo(url) &&
+    imageSizeInfo(url).width > MIN_GALLERY_IMG_WIDTH
```

Comment:
optional: we could do the following:

```suggestion
  const included =
    !get(node, 'data.excludeFromGallery', false) &&
    imageSizeInfo(url) &&
    imageSizeInfo(url).width > MIN_GALLERY_IMG_WIDTH
  if (!included) {
    return
  }
```

and then just `.filter(Boolean)` instead of `.filter(i => i.included)`

Reply:
I thought about that but it felt less clear in term of what the code's doing.